### DISCUSSION TITLE
x64dbg loads wrong address space

### DISCUSSION THREAD

**Discussion by karaok1** [x64dbg/x64dbg#3060] [Q&A]
x64dbg is loading the wrong address ranges I think. When I try to locate a memory address which I found in CheatEngine, it can't find the address and even if it finds an address, it is a wrong one.
What I tried:
Tried to navigate using the file offset, but it doesn't navigate to the right place.
Tried to navigate from the Disassembly view using file offset, but it wasn't the right address either.

It is a x64 Console application that I try to debug. Even the address ranges don't match with CheatEngine, 

![x64dbg_tj4coc5A4O](https://user-images.githubusercontent.com/17222122/229533257-695da54e-1883-47fd-9269-48119c430394.jpg)

**Comment by mrexodia** [comment]
This is called ASLR.
